src/App/Fossa/Analyze/Types.hs

```diff
@@ -113,9 +113,20 @@ data DiscoveredProjectIdentifier = DiscoveredProjectIdentifier
   }
   deriving (Eq, Ord, Show)
 
+{-# WARNING analyzeProject' "analyzeProject' has been renamed to analyzeProjectStaticOnly. Please use that name for all future implementations." #-}
 class AnalyzeProject a where
-  -- | Analyze a project with any tactic.
+  -- | Analyze a project with any tactic including dynamic ones.
   analyzeProject :: AnalyzeTaskEffs sig m => FoundTargets -> a -> m DependencyResults
 
   -- | Analyze a project with only static tactics.
-  analyzeProject' :: AnalyzeStaticTaskEffs sig m => FoundTargets -> a -> m DependencyResults
+  -- Static here generally means not using any external utilities.
+  -- An exception is berkeleydb which calls a program embedded in the CLI.
+  analyzeProjectStaticOnly :: AnalyzeStaticTaskEffs sig m => FoundTargets -> a -> m DependencyResults
```

Comment:
I think we should be very precise: static concretely means "statically analyzing". Whether we execute external tools is not relevant to this equation.